verilog/tools/ls/BUILD

```diff
@@ -62,13 +62,27 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "symbol-table-handler",
+    srcs = ["symbol-table-handler.cc"],
+    hdrs = ["symbol-table-handler.h"],
+    deps = [
+        "//verilog/analysis:symbol_table",
```

Comment:
Some dependencies are missing; all dependencies that we use in headers included in `symbol-table-handler.{h,cc}` should be mentioned here
```
"@com_google_absl//absl/container:flat_hash_set",
"@com_google_absl//absl/flags:flag",
"@com_google_absl//absl/strings",
"//common/lsp:lsp-protocol",
"//common/strings:line_column_map",
"//common/util:file_util",
"//verilog/analysis:symbol_table",
"//verilog/analysis:verilog_filelist",
"//verilog/analysis:verilog_project",
```
Also, `"@com_google_absl//absl/status:statusor",`, but I am not sure if we actually need that (it is included, but never used)

Reply:
Then you might also need to add `"//verilog/tools/ls:__pkg__",` to the visibility list in `//common/strings:line_column_map` .